vennthandler.py

```diff
@@ -76,13 +76,26 @@ def do_OPTIONS(self):
         self.send_header('Access-Control-Allow-Methods',
                          'OPTIONS, GET, HEAD, POST')
         self.send_header('Allow', 'OPTIONS, GET, HEAD, POST')
+        self.send_header('Access-Control-Max-Age', '86400')
```

Comment:
Note: Should hopefully allow browsers to cache this preflight request, which should help reduce the number of API requests we hit the server with.